includes/revisions-control/class-revisions-control.php

```diff
@@ -0,0 +1,136 @@
+<?php
+/**
+ * Handles the revisions control for Newspack, managing how many revisions should be kept in the database
+ *
+ * @package Newspack
+ */
+
+namespace Newspack;
+
+use DateTime;
+use WP_Post;
+/**
+ * Revisions Control class
+ */
+class Revisions_Control {
+
+	/**
+	 * Flag to make sure hooks are initialized only once
+	 *
+	 * @var boolean
+	 */
+	private static $initiated = false;
+
+	/**
+	 * Initializes the hook
+	 *
+	 * @return void
+	 */
+	public static function init() {
+		if ( ! self::$initiated ) {
+			add_filter( 'wp_revisions_to_keep', [ __CLASS__, 'filter_revisions_to_keep' ] );
+			add_filter( 'pre_delete_post', [ __CLASS__, 'pre_delete_revision' ], 10, 2 );
+			self::$initiated = true;
+		}
+	}
+
+	/**
+	 * Gets the option stored in the database with the feature configuration
+	 *
+	 * @return array
+	 */
+	public static function get_option() {
```

Comment:
totally agree and usually I do this. I guess I didn't here because I was writing tests and didn't want to deal with it at the time. But upgraded code and tests to make methods private